src/react/components/TextEditor/StyleButton.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+export class StyleButton extends React.Component {
```

Comment:
Use funtion components for consistency. Everything else under src/react/components are functional components.